docs/framework/install/on-windows-and-server.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Install .NET Framework on Windows
+description: Learn how to install .NET Framework on Windows 11, Windows 10, and Windows Server. This article also includes information about .NET Framework and unsupported versions of Windows, such as Windows 8, Windows Vista, and Windows XP.
+ms.date: 02/12/2025
+---
+# Install .NET Framework on Windows and Windows Server
+
+.NET Framework is included on all current versions of Windows and Windows Server. This article helps you understand which version of .NET Framework is included in Windows and Windows Server, and if an upgrade is available.
+
+The latest version of .NET Framework is 4.8.1. This version of .NET Framework supports all .NET Framework 4 apps, even if that app was created for an older version of .NET Framework 4. For more information about a specific release of .NET Framework, see [.NET Framework versions and dependencies](versions-and-dependencies.md).
```

Comment:
What does "an older version of .NET Framework 4" mean?

Reply:
Would it be clearer with Older release? Prior release? Prior version? You can't and don't need to downgrade the version of .NET Framework on your machine to run an app that targets an older version.

Reply:
Sure. I'll trim this and simplify it